src/pybind/mgr/dashboard/frontend/src/app/core/navigation/navigation/navigation.component.html

```diff
@@ -225,6 +225,11 @@
                             i18n-title
                             *ngIf="permissions.nfs.read && enabledFeature.nfs"
                             class="tc_submenuitem tc_submenuitem_rgw_nfs"><span i18n>NFS</span></cds-sidenav-item>
+          <cds-sidenav-item route="/rgw/configuration"
+                            [useRouter]="true"
+                            title="Configuration"
+                            i18n-title
+                            class="tc_submenuitem tc_submenuitem_rgw_configuration"><span i18n>Configuration</span></cds-sidenav-item>
```

Comment:
i see you have added the test class here which is a god thing, but you are not updating the unit test or e2e test files.

Reply:
Added the e2e